src/stan_math_backend/Transform_Mir.ml

```diff
@@ -68,31 +69,22 @@ let rec switch_expr_to_opencl available_cl_vars (Expr.Fixed.({pattern; _}) as e)
          (Expr.Typed.adlevel_of (List.nth_exn args i))
```

Comment:
consider pulling out something like `let arg = List.nth_exn args i in` since it's used twice? not a big deal